src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -2170,6 +2170,10 @@ public partial class DbEnumerator : System.Collections.IEnumerator
     }
     public abstract partial class DbException : System.Runtime.InteropServices.ExternalException
     {
+        /// <summary>
```

Comment:
Right now, documentation is added separately in https://github.com/dotnet/dotnet-api-docs and not here - that's why nothing currently has XML documentation. So let's remove this for now (from ref as well). Once this PR is merged we can do a separate PR to that repo, and if one day documentation is added here, we can do it for all the System.Data docs together.